doc/tutorials.rst

```diff
@@ -20,37 +60,119 @@ plugins, and devices, via simple and easy-to-follow examples.
+
+
+.. toctree::
+    :hidden:
+    :maxdepth: 1
+
+    tutorials/pennylane_plugins_hybrid
+    tutorials/pennylane_pytorch_noise
+    tutorials/pennylane_run_state_preparation
+
 .. customgalleryitem::
     :tooltip: Use quantum machine learning in a multi-device quantum algorithm.
     :figure: ../examples/figures/photon_redirection.png
     :description: :ref:`plugins_hybrid`
 
-.. customgalleryitem::
-    :tooltip: Multiple expectation values, Jacobians, and keyword arguments.
-    :description: :ref:`advanced_features`
-
 .. customgalleryitem::
     :tooltip: Extend PyTorch with real quantum computing power.
     :figure: ../examples/figures/bloch.gif
     :description: :ref:`pytorch_noise`
 
+.. customgalleryitem::
+    :tooltip: Do arbitrary state preparation on a real quantum computer.
+    :figure: ../examples/figures/state_prep.png
+    :description: :ref:`state_preparation`
+
+
 :html:`<div style='clear:both'></div>`
 
 
-:html:`<h3>Quantum machine learning with PennyLane</h3>`
+Ready-to-use templates
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+PennyLane provides a growing library of ready-to-use templates of common quantum machine learning circuit architectures and embedding functions. These can be used to easily embed classical data and build (and train) complex quantum machine learning models. They are provided as functions that can be called inside QNodes; for details see :mod:`pennylane.templates`. In the tutorials below, we will go through a couple of examples that use these templates in detail.
+
 
-Take a deeper dive into quantum machine learning by exploring cutting-edge
-algorithms using PennyLane and near-term quantum hardware.
+.. toctree::
+    :hidden:
+    :maxdepth: 2
+
+    tutorials/pennylane_run_templates_DV
+    tutorials/pennylane_templates_CV
 
 .. customgalleryitem::
-    :tooltip: Do arbitrary state preparation on a real quantum computer.
-    :description: :ref:`state_preparation`
+    :tooltip: Make a strongly entangling circuit in a single line of code.
+    :figure: ../examples/figures/slsec_example.png
+    :description: :ref:`templates_DV`
+
+.. customgalleryitem::
+    :tooltip: Make a Quantum Neural Network in a single line of code.
+    :figure: ../examples/figures/cvqnn_example.png
+    :description: :ref:`templates_CV`
+
+:html:`<div style='clear:both'></div>`
+
+Quantum machine learning algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PennyLane is desgined to do machine learning with quantum computers. These examples show PennyLane's implementation of some of the well-known algorithms from the recent research in quantum machine learning. This is, however, just to showcase how quantum machine learning with PennyLane *could* look like. In principle *any* technique where a quantum circuit is optimized and potentially run on a real device can be implemented in PennyLane.  
```

Comment:
Just so we don't limit ourselves. Maybe in future optimisation without ML will become more important.

```suggestion
One goal of PennyLane is to enable machine learning with quantum computers. These examples show PennyLane's implementation of some of the well-known algorithms from the recent research in quantum machine learning. This is, however, just to showcase how quantum machine learning with PennyLane *could* look like. In principle *any* technique where a quantum circuit is optimized and potentially run on a real device can be implemented in PennyLane.  
```